enhancements/network/on-prem-service-load-balancers.md

```diff
@@ -0,0 +1,370 @@
+---
+title: on-prem-service-load-balancers
+authors:
+  - "@russellb"
+reviewers:
+  - TBD
+approvers:
+  - TBD
```

Comment:
I don't know.  I wasn't sure, left it blank, and forgot to come back to it.

some thoughts ...

- someone from architecture team (you if interested, @derekwaynecarr ?  or @smarterclayton indicated he has a lot of interest in this topic)
- networking group lead (@knobunc )
- someone from team likely to own this (kni networking @celebdor 's team)
- lead from networking edge team, since they own service load balancers more generally

Reply:
since @danwinship already commented, maybe you'd have interest in approving this as a representative from the networking group?

Reply:
... and it occurs to me there's no guidance on how to fill out reviewers/approvers in the template, so i'll follow up on that separatement

Reply:
sure

Reply:
I've updated this to include names for reviewers and approvers.  I've listed everyone who has commented so far as a reviewer.  I listed @knobunc , @danwinship , @danehans as approvers as representatives from different aspects of openshift networking.

I think this is fine to merge now if there are no new concerns.  Note that it's still marked "proposed", as I'd expect another enhancement to be filed to refine the operator plans.